gettsim/social_insurance_contributions/ges_krankenv.py

```diff
@@ -335,10 +335,39 @@ def _ges_krankenv_midi_job_sum_arbeitn_arbeitg_m(
     return out
 
 
-def _ges_krankenv_midi_job_arbeitg_m(
+def _ges_krankenv_midi_job_sum_arbeitn_arbeitg_m_ab_2022(
```

Comment:
I very much like that we have separate functions for `arbeitn`, `arbeitg` and `arbeitn_arbeitg` in `ges_krankenv`. Can you do the same for rentenv, arbeitsl_v and pflegev (If it makes sense there)?